src/services/findAllReferences.ts

```diff
@@ -616,7 +616,8 @@ namespace ts.FindAllReferences {
             }
 
             const checker = program.getTypeChecker();
-            const symbol = checker.getSymbolAtLocation(node);
+            // constructors should use the class symbol, detected by name, if present

```

Comment:
that's because, currently, constructor declarations *do* have a symbol, but

1. it's useless for find-all-refs
2. getSymbolAtLocation doesn't return it anyway.